test/util.test.ts

```diff
@@ -46,4 +46,45 @@ describe('findManifests', () => {
     //Bad directory name (non-existent) should yield an empty array
     expect(noFilesEither).toEqual([]);
   });
+});
+
+test('derive file name from url', () => {
+  const devUrl = 'github:crossplane/crossplane@0.14.0';
+  const localFile = './foo/bar/baz/fooz.yaml';
```

Comment:
Add more scenarios:

- local file that ends with `.yml` (its also a common YAML extension)
- regular URL (not `github:`)